_announcements/week-3.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Week 3 Announcement
+week: 3
+date: 2022-2-13
+---
+
+Hello everyone! Welcome to the Linux Sysadmin Decal! Here are your week 3 announcements:
+
+1. The third week’s labs and lectures have been released! You can view them below.
+2. Due to maintenance of OCF servers on Saturday, Lab 2 is now due Monday, 2/14 at 11:59 pm.
```

Comment:
maybe add one about the fact that vm's are distributed and login instructions were sent over email>